packages/fs-detectors/README.md

```diff
@@ -0,0 +1,27 @@
+# @vercel/routing-utils
```

Comment:
This README still needs to be updated, I think 👀

Reply:
I realized we dont have readmes for build-utils or any builders so I'll just remove it. This utility isn't really meant for public consumption. [1f84f9d](https://github.com/vercel/vercel/pull/8054/commits/1f84f9d127219e715e97720dada8b2aabc17c3b8)